autoload/fzf/vim.vim

```diff
@@ -53,7 +53,8 @@ function! s:bash()
 
   let custom_bash = s:conf('preview_bash', '')
   let git_bash = 'C:\Program Files\Git\bin\bash.exe'
-  let candidates = filter(s:is_win ? [custom_bash, 'bash', git_bash] : [custom_bash, 'bash'], 'len(v:val)')
+  let scoop_git_bash = exists('$GIT_INSTALL_ROOT') ? '$GIT_INSTALL_ROOT' . '\bin\bash.exe : ''
```

Comment:
looks like single quote is missing here `'\bin\bash.exe : ''`

Reply:
```
Error detected while processing function fzf#vim#with_preview[13]..<SNR>78_bash:
line    7:
E115: Missing quote: '\bin\bash.exe : ''
```

Reply:
@junegunn ^

Reply:
Oops, thanks!

Reply:
Sorry, this happened because I was making changes to the local fzf.vim plugin and my fork, which I hadn't synced.